test/end-to-end/pages/payment/continueCancel.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const testConfig = require('test/config');
+
+module.exports = function () {
+    const I = this;
+    I.wait(3);
+    I.waitForText('Your payment has been cancelled', testConfig.TestWaitForTextToAppear);
+    I.seeInCurrentUrl(testConfig.TestGovUkCardPaymentsUrl);
+    I.waitForElement('#return-url', testConfig.TestWaitForElementToAppear);
```

Comment:
is it possible to move ids to a constant value so that it will be clear for us to see what ids or fields we are testing  when we come to this file at later stage.